cmd/celestia-appd/cmd/root.go

```diff
@@ -149,6 +150,9 @@ func initRootCmd(rootCmd *cobra.Command, encodingConfig encoding.Config) {
 		keys.Commands(app.DefaultNodeHome),
 		bscmd.VerifyCmd(),
 	)
+	rootCmd.AddCommand(
+		snapshot.Cmd(NewAppServer),
+	)
```

Comment:
nit: I guess this could have just been appended to the `AddCommand` function above

Reply:
true. TBH the contents of this function don't make sense to me. IDK why we add some commands in a block at the top and then add more later

Reply:
yeah it's all a bit messy but not too problematic